### PR TITLE
Update README

### LINKED ISSUE #4: Update readme
Needs updated build instructions, remove discussion of acquire drivers, etc.

### PR DESCRIPTION
**Pull request by aliddell**
Closes #4.

- Update build instructions
- Update examples
- Remove references to acquire drivers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,328 +1,126 @@
...
+# Acquire Zarr streaming library
 
-[![Build](https://github.com/acquire-project/acquire-driver-zarr/actions/workflows/build.yml/badge.svg)](https://github.com/acquire-project/acquire-driver-zarr/actions/workflows/build.yml)
-[![Tests](https://github.com/acquire-project/acquire-driver-zarr/actions/workflows/test_pr.yml/badge.svg)](https://github.com/acquire-project/acquire-driver-zarr/actions/workflows/test_pr.yml)
+[![Build](https://github.com/acquire-project/acquire-zarr/actions/workflows/build.yml/badge.svg)](https://github.com/acquire-project/acquire-zarr/actions/workflows/build.yml)
+[![Tests](https://github.com/acquire-project/acquire-zarr/actions/workflows/test.yml/badge.svg)](https://github.com/acquire-project/acquire-zarr/actions/workflows/test_pr.yml)
 [![Chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://acquire-imaging.zulipchat.com/)
 
-This is an Acquire Driver that supports chunked streaming to [zarr][].
+This library supports chunked, compressed streaming to [Zarr][].
```
Comment (nclack) [inline README.md:7]: and multiscale to OME-Zarr :tada:

File: README.md [hunk 2]
```diff
@@ -1,328 +1,126 @@
...
-  DimensionType_Time, // type of the dimension
-  0,   // append dimension; we don't know the full size a priori
-  100, // number of pixels in a chunk
-  1);  // one 100-timepoint chunk per shard along this dimension
+```bash
+cmake --build /path/to/build
 ```
 
-### Compression
+## Usage
```
Comment (nclack) [inline README.md:72]: Is there an example for the python api somewhere?

File: README.md [hunk 3]
```diff
@@ -1,328 +1,126 @@
...
+A typical use case for a 4-dimensional acquisition might look like this:
 
 ```c
-int
-storage_properties_set_enable_multiscale(struct StorageProperties* out,
-                                         uint8_t enable);
+ZarrStreamSettings settings;
+settings.store_path = "my_stream.zarr";
+settings.data_type = ZarrDataType_uint16;
+settings.version = ZarrVersion_2;
```
Comment (nclack) [inline README.md:84]: Why not have a Zarr v3 example?

File: README.md [hunk 4]
```diff
@@ -1,328 +1,126 @@
...
+
+settings.dimensions[2].name = "y";
+settings.dimensions[2].type = ZarrDimensionType_Space;
+settings.dimensions[2].array_size_px = 1080; // height
+settings.dimensions[2].chunk_size_px = 270; // 4 x 4 tiles of size 270 x 480
+
+settings.dimensions[3].name = "x";
+settings.dimensions[3].type = ZarrDimensionType_Space;
+settings.dimensions[3].array_size_px = 1920; // width
+settings.dimensions[3].chunk_size_px = 480; // 4 x 4 tiles of size 270 x 480
```
Comment (nclack) [inline README.md:105]: using [designated initializers](https://en.cppreference.com/w/cpp/language/aggregate_initialization#Designated_initializers) might enhance readability here.

File: README.md [hunk 5]
```diff
@@ -1,328 +1,126 @@
...
+size_t bytes_written;
+ZarrStream_append(stream, my_frame_data, my_frame_size, &bytes_written);
+assert(bytes_written == my_frame_size);
 ```
 
-To enable, pass `1` as the `enable` parameter, and to disable, pass `0`.
-
-If enabled, the Zarr writer will write a pyramid of frames, with each level of the pyramid halving each dimension of
-the previous level, until the dimensions are less than or equal to a single tile.
+Look at acquire.zarr.h for more details.
```
Comment (nclack) [inline README.md:114]: I believe github allows relative links in readmes.
```suggestion
Look at [acquire.zarr.h](include/acquire.zarr.h) for more details.
```

File: vcpkg.json [hunk 6]
```diff
@@ -2,7 +2,7 @@
   "dependencies": [
     {
       "name": "blosc",
-      "version>=": "1.21.5"
+      "version>=": "1.21.6"
```
Comment (jeskesen) [inline vcpkg.json:5]: is there a reason you're upgrading the blosc version requirement as part of a README change?

File: README.md (hunk 2 above)
Comment (aliddell) [reply README.md:72]: There is now.

**Review by jeskesen — COMMENTED** [review]
Looks good, except for the version upgrade that got put in. 

I notice that there are comments by @nclack that should probably be resolved before approval.

File: vcpkg.json (hunk 6 above)
Comment (aliddell) [reply vcpkg.json:5]: Matching my environment with the documentation and the requirements. The previous release was over a year ago.

**Review by jeskesen — APPROVED** [review]

**Pull request merged**